### PR TITLE
FS-3147

### PR DESCRIPTION
**Pull request by robk-dluhc**
https://dluhcdigital.atlassian.net/browse/FS-3147
Create image when checking in code

### Change description
Create image when tagging or pushing code so the images are available for deployment.

- [ ] Unit tests and other appropriate tests added or updated
- [ ] README and other documentation has been updated / added (if needed)
- [ ] Commit messages are meaningful and follow good commit message guidelines (e.g. "FS-XXXX: Add margin to nav items preventing overlapping of logo")

### How to test
_If manual testing is needed, give suggested testing steps_

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -8,4 +8,4 @@ COPY . .
 
 EXPOSE 8080
 
-CMD bash -c "flask db upgrade && gunicorn -w 1 -b 0.0.0.0:8080 app:create_app()"
+CMD ["gunicorn", "-w", "1", "-b", "0.0.0.0:8080", "app:create_app()"]
```
Comment (aaronwilliamsv1) [inline Dockerfile:11]: This shouldn't be needed, or changed?

We don't want to mess with the Dockerfile, as that's used by both processes - it should not needed to be changed at all to allow it to run for paketo

**Comment by aaronwilliamsv1** [comment]
`Commit messages are meaningful`

I'd suggest doing a quick squash on this one, there's a lot of back and forth that won't be relevant to history

File: Dockerfile (hunk 1 above)
Comment (robk-dluhc) [reply Dockerfile]: Removed

**Comment by aaronwilliamsv1** [comment]
> `Commit messages are meaningful`
> 
> I'd suggest doing a quick squash on this one, there's a lot of back and forth that won't be relevant to history

Still relevant

**Review by aaronwilliamsv1 — APPROVED** [review]
LGTM

**Pull request merged**